### PR TITLE
A tiny js world

### PR DESCRIPTION
**Pull request by kasionio**
### a-tiny-js-world

[Demo](https://kasionio.github.io/a-tiny-JS-world)
[Code base](https://github.com/kasionio/a-tiny-JS-world/blob/gh-pages/index.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/kasionio/a-tiny-JS-world/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ print('<strong>ABC</strong>', 'div');
+
+ print('human; John; male; 2; 2; Hello world!; Rex, Tom, Jenny');
+ print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny');
+ print('human; <strong>John</strong>; male; 2; 2; <em>Hello world!</em>; Rex, Tom, Jenny', 'div');
+ */
+
+const message = (arr, props) => arr.map(item => props.map(prop => item[prop])
+  .join('; '))
+  .join('\n');
```
Comment (OleksiyRudenko) [inline submissions/kasionio/a-tiny-JS-world/index.js:79]: `arr` and `item` are too generic names. Generic names are good for examples.

In real code keep names as specific as only possible. 

I assume these are `inhabitants` and `inhabitant`?

**Review by OleksiyRudenko — APPROVED** [review]
@kasionio well done. Approved with reservations

File: submissions/kasionio/a-tiny-JS-world/index.js (hunk 1 above)
Comment (kasionio) [reply submissions/kasionio/a-tiny-JS-world/index.js:79]: Thanks! It's easy to understand for me how the code works with generic names, but now I get it. 

Yes, in this case `arr` are `inhabitants` and `item` is 'inhabitant'

Comment (OleksiyRudenko) [reply submissions/kasionio/a-tiny-JS-world/index.js:79]: Imagine you come across a code base with hundreds lines of code and dozens of modules. The closer code to plain English with domain specificity the easier to read it, mainitain and scale.

Comment (kasionio) [reply submissions/kasionio/a-tiny-JS-world/index.js:79]: Thanks for the explanation. Now I realize

**Pull request merged**